src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -120,6 +121,19 @@ export const WorkspaceData = (signal) => ({
     );
     return res.json();
   },
+  updateRecord: async (
+    root: string,
+    instanceId: string,
+    recordType: string,
+    recordId: string,
+    attributes: object
```

Comment:
This type could be more descriptive than `object`.
```suggestion
    attributes: { [attribute: string]: any }
```